.github/workflows/main.yml

```diff
@@ -48,4 +48,24 @@ jobs:
       - name: Run basic yaml validation
         run: |
           python tests/validate_yaml.py
-        
+  on-fail:
```

Comment:
I believe this change belongs to https://github.com/kernelci/kernelci-pipeline/pull/712 which was already reverted in https://github.com/kernelci/kernelci-pipeline/pull/725

Reply:
Removed, got them here by mistake

Reply:
Removal of this workflow change also reverted all the other changes applied during review.

I choose to believe it might have been done by mistake. If that's not the case, it's a next-level review trolling :laughing:

Reply:
Resolved in: https://github.com/kernelci/kernelci-pipeline/pull/734